mmv1/products/compute/Subnetwork.yaml

```diff
@@ -225,7 +225,10 @@ properties:
       A subnetwork with purpose set to `PRIVATE_NAT` is used as source range for Private NAT gateways.
       Note that `REGIONAL_MANAGED_PROXY` is the preferred setting for all regional Envoy load balancers.
       If unspecified, the purpose defaults to `PRIVATE`.
-    immutable: true
+    update_verb: 'PATCH'
+    update_uri: 'projects/{{project}}/regions/{{region}}/subnetworks/{{name}}'
+    update_id: 'purpose'
+    fingerprint_name: 'fingerprint'
```

Comment:
All you need to do to make the field updatable (if it uses the same API as the resource in general) is remove `immutable`. update_verb etc. are only needed if the resource needs to make an additional call to a separate endpoint to do the update. See https://googlecloudplatform.github.io/magic-modules/reference/field/#update_url for more information.

```suggestion
```